### PR TITLE
Improve load options

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,16 +25,32 @@ export {
...
+  // the full mediator URL)
+  let mediatorUrl;
+  if(typeof options === 'string') {
+    mediatorUrl = options;
+  } else if(options && typeof options === 'object' &&
+    typeof options.mediatorOrigin === 'string') {
+    mediatorUrl = `${options.mediatorOrigin}/mediator`;
+  } else {
+    throw new Error(
+      '"options.mediatorOrigin" must be a string express the ' +
```
Comment (davidlehn) [inline index.js]: Should that be "expressing"?  Might just reword to say "options.mediatorOrigin" is missing.

Comment (davidlehn) [reply index.js]: I read the code wrong, maybe it can just say that param needs to be a string.

File: index.js [hunk 2]
```diff
@@ -15,8 +15,7 @@ import {CredentialsContainer} from './CredentialsContainer.js';
 import {PermissionManager} from './PermissionManager.js';
 import {WebCredential} from './WebCredential.js';
 
-const DEFAULT_MEDIATOR = 'https://authn.io/mediator' + '?origin=' +
-  encodeURIComponent(window.location.origin);
```
Comment (davidlehn) [inline index.js:19]: Is that origin param no longer needed?  Doesn't look like it's added back in this patch?

File: index.js [hunk 3]
```diff
@@ -26,16 +25,32 @@ export {
...
   loaded = true;
-  return load(mediatorUrl);
+  return load(options);
 }
 
-export async function load(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function load(options = {
+  mediatorOrigin: DEFAULT_MEDIATOR_ORIGIN
+}) {
+  // backwards compatibility (`options` used to be a string for expressing
```
Comment (davidlehn) [inline index.js:40]: Would it be too disruptive to just drop old support and do a breaking release?

File: index.js [hunk 4]
```diff
@@ -26,16 +25,32 @@ export {
...
   }
 
   loaded = true;
-  return load(mediatorUrl);
+  return load(options);
 }
 
-export async function load(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function load(options = {
+  mediatorOrigin: DEFAULT_MEDIATOR_ORIGIN
```
Comment (davidlehn) [inline index.js:38]: Should have some jsdocs and README docs for `loadOnce` and `load` and the options.

File: index.js (hunk 1 above)
Comment (dlongley) [reply index.js]: I think it's better to be more descriptive here. I fixed the typo.

File: index.js (hunk 2 above)
Comment (dlongley) [reply index.js:19]: Yeah, it's no longer needed. This patch is, in part, to help remove that.

File: index.js (hunk 3 above)
Comment (dlongley) [reply index.js:40]: Yeah, in this release and at this time. We can do a breaking one later if we feel we need to.

File: index.js (hunk 4 above)
Comment (dlongley) [reply index.js:38]: Indeed, but would be too low on my priority list for me to take that action at this time. PRs welcome :)

**Pull request merged**